helper/keysutil/lock_manager.go

```diff
@@ -512,5 +516,9 @@ func (lm *LockManager) getStoredPolicy(ctx context.Context, storage logical.Stor
 		return nil, err
 	}
 
+	if &policy != nil {
```

Comment:
Can this ever happen? Will DecodeJSON actually nil out a stack-declared value if something goes wrong? I thought it just won't fill in the entries.